wuvt/templates/base.html

```diff
@@ -142,6 +142,30 @@ <h1>
 <script src="{{ url_for('static', filename='js/stream.js', v=3) }}"></script>
 <script src="{{ url_for('static', filename='js/playlists_by_date.js') }}"></script>
 <script src="{{ url_for('init_js') }}"></script>
+<script>
```

Comment:
not critical now, but before we go prod with this, we should not inline the script.